### PR TITLE
LPS-100140, LPS-100143

### PR DESCRIPTION
**Pull request by julien**

### CODE REVIEW

**Comment by julien** [comment]
ci:test:sf

**Comment by julien** [comment]
ci:test:relevant

**Comment by julien** [comment]
ci:test:sf

**Comment by julien** [comment]
ci:test:sf

**Comment by ambrinchaudhary** [comment]
Just started reviewing :)

File: modules/apps/asset/asset-taglib/src/main/resources/META-INF/resources/asset_categories_selector/AssetVocabularyCategoriesSelector.es.js [hunk 1]
```diff
@@ -12,300 +12,279 @@
...
+					}
+				}),
+				p_auth: Liferay.authToken
+			}),
+			credentials: 'include',
+			method: 'POST',
+			'x-csrf-token': Liferay.authToken
+		},
+		link: `${window.location.origin}${themeDisplay.getPathContext()}
+				/api/jsonws/invoke`
```
Comment (wincent) [inline modules/apps/asset/asset-taglib/src/main/resources/META-INF/resources/asset_categories_selector/AssetVocabularyCategoriesSelector.es.js]: This still has bad whitespace (newline and tabs in the middle of the link text).

**Comment by julien** [comment]
Just started reviewing :)

**Comment by julien** [comment]
Forwarded to https://github.com/carloslancha/liferay-portal/pull/216

**Comment by julien** [comment]
ci:close

**Pull request closed without merging**